docs/DOCKER.rst

```diff
@@ -280,3 +280,50 @@ For instance, you can modify these lines in the dockerfile:
 
     USER scality
     ENTRYPOINT ["/usr/src/app/docker-entrypoint.sh"]
+
+Continuous integration with Docker hosted CloudServer
+-----------------------------------------------------
+
+When you start the Docker Scality Zenko CloudServer image, you can adjust the
+configuration of the Scality Zenko CloudServer instance by passing one or more
+environment variables on the docker run command line.
+
+Sample ways to run it for CI are:
+
+- With custom locations, including one hosted on AWS, and custom credentials mounted:
+
+.. code:: shell
+
+    docker run --name CloudServer -p 8000:8000
```

Comment:
I think it's confusing to use Cloudserver since our docker image is s3server.

Reply:
I thought we were not allowed to use s3server for new content anymore? Plus, eventually, the image should be renamed cloudserver, and the whole documentation talks about cloudserver, so I guess it should be OK?